SonarQube.Common/Serializer.cs

```diff
@@ -37,7 +37,7 @@ public static class Serializer
 
             // Serialize to memory first to reduce the opportunity for intermittent
             // locking issues when writing the file
-            using (MemoryStream stream = new MemoryStream())
+            MemoryStream stream = new MemoryStream();
```

Comment:
Not sure I like this rule. It assumes that the StreamWriter will dispose it's encapsulated writer. I'm sure it does, but it would still be safer to call dispose twice....